include/d/d_com_inf_game.h

```diff
@@ -2194,6 +2212,10 @@ inline void dComIfGp_setItemBeastNumCount(int i_idx, s16 num) {
     g_dComIfG_gameInfo.play.setItemBeastNumCount(i_idx, num);
 }
 
+inline s16 dComIfGp_getItemBeastNumCount(int i_idx) {
+    g_dComIfG_gameInfo.play.getItemBeastNumCount(i_idx);
```

Comment:
missing return (if you didn't know, you can check this in the build output, there are a bunch of "return value expected" warnings)